docs/inventory/README.md

```diff
@@ -1,64 +1,48 @@
 # Inventory
 This is the Inventory component of Forseti Security tool.
-
+  * [Pre-requisites](#pre-requisites)
+  * [Executing inventory](#executing-inventory)
+  * [Developing on inventory](#developing-on-inventory)
+    * [Collecting and storing new data with inventory](#collecting-and-storing-new-data-with-inventory)
+  
 ## Pre-requisites
 See the [PREREQUISITES](/docs/prerequisites/README.md) guide.
 
-## How to Run
+## Executing inventory
 After running setup.py, as long as your virtualenv is activated, then you can be in
 any directory to invoke the console script:
 
 ```sh
 $ forseti_inventory
 ```
 
-You can also use the convenience [dev_inventory.sh script](/scripts) to run
-forseti_inventory. Make a copy of dev_inventory.sh.sample
-as dev_inventory.sh, edit the script for the appropriate
-commandline flags, and invoke the script from the repo root to run inventory.
+You can also use the convenience [dev_inventory.sh script](/scripts) to run forseti_inventory. Make a copy of dev_inventory.sh.sampleas dev_inventory.sh, edit the script for the appropriate commandline flags, and invoke the script from the repo root to run inventory.
 
 ```sh
 $ cd path/to/forseti-security
 $ scripts/dev_inventory.sh
 ```
+## Developing on inventory
+### Collecting and storing new data with inventory
 
-## How to Test
-Look at the test instructions in the tests [README](/docs/tests/README.md).
-
-## How to add a new resource to the inventory
-
-It's straightforward to add a new resource to inventory for collection.
-This workflow is designed to be generic across resource types.
-
-The main steps are:
-
-1. Define the database table schema to store the resource data.
-
-2. Create a new pipeline worker to process the new table and the table fieldnames.
+It's straightforward to add a new resource to inventory for collection. This workflow is designed to be generic across resource types.
 
-3. Write a transformation to flatten the data returned by GCP APIs into a format
-that's ingestiable.
+1. Define a new table schema for the 'flattended' data you need to store.
+Each relevant field of data you retrieve from an API should correspond to a column in the table schema.
 
-Look at this PR for an [end-to-end example loading of a resource].
+2. Define a new table schema for the 'raw data you need to store.
+Additionally, we aim to store the raw API data as json to assist troubleshooting.
 
-Step-by-Step:
+Once you've completed these steps create a Pull Request ([example](https://github.com/GoogleCloudPlatform/forseti-security/pull/159)).
```

Comment:
I would link to the official Github documentation on how to create a pull request, and not an actual pull request.